### DISCUSSION TITLE
Transport priority qos still not being updated?

### DISCUSSION THREAD

**Discussion by guni9191** [eProsima/Fast-DDS#5352] [Ideas]
For industrial applications, this is critical to use transport priority since emergency topics should be sent at most priority. 
I'm using ROS2 since foxy, and still frustrated to see that this qos is not implemented.
Does this concept use DSCP markings? then wouldn't it be too burdensome to implement this concept?

**Comment by JesusPoderoso** [comment]
Hi @guni9191, sorry for the inconvenience.
Fully supporting the transport priority qos is not part of our current roadmap. If you are interested in speeding this development up for future releases, do not hesitate to contact our commercial support [support@eprosima.com](mailto:support@eprosima.com).

On the other hand, if you are interested on providing some priority to some specific elements that take part of the DDS communications, perhaps the [flow controllers](https://fast-dds.docs.eprosima.com/en/stable/fastdds/use_cases/large_data/large_data.html#flow-controllers) suit your needs. They allow, for instance, define some [priority and bandwidth](https://fast-dds.docs.eprosima.com/en/stable/fastdds/property_policies/flow_control.html) to the DataWriters, and customize low-level [thread settings](https://fast-dds.docs.eprosima.com/en/stable/fastdds/dds_layer/core/policy/eprosimaExtensions.html#threadsettingsqos) through their [QoS configuration](https://fast-dds.docs.eprosima.com/en/stable/fastdds/dds_layer/core/policy/eprosimaExtensions.html#flowcontrollersqos).
